spec/requests/api/v1/list_restaurants_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::ListRestaurantsController, type: :request do
+  describe 'GET /v1/list_restaurants' do
+    it 'should return all the restaurants' do
+      create(:restaurant)
+      get '/api/v1/list_restaurants'
+      json_response = JSON.parse(response.body)
+      expect(response.status).to eq 200
+      expect(json_response['entries'].first['name']).to eq "Nisses"
+    end
```

Comment:
This request spec should have an actual `expected_response` that lists a JSON response from the endpoint with one restaurant. You are deciding how you want to output your response, then making the code do that. How should this data be formatted to make it easy for the client / your front end team to consume it?